js/apps/admin-ui/src/identity-providers/IdentityProvidersSection.tsx

```diff
@@ -92,18 +93,31 @@ export default function IdentityProvidersSection() {
 
   useFetch(
```

Comment:
This fetch is now done twice, once here, and another time in the loader. We should remove this one.